_posts/guides/2019-08-23-guide_install-css-assets.md

```diff
@@ -7,4 +7,42 @@ tags:
   - cs:s
 ---
 
-TODO: Show how to install CS:S assets from SteamCMD
+
+This guide covers how to install CS:S assets using SteamCMD.
+## Prerequisites 
+- An internet connection.
+- Some free space on your PC. (~4GB)
+
+## Getting SteamCMD
+For Linux/Mac refer to Valve's guide:
+https://developer.valvesoftware.com/wiki/SteamCMD
+### Windows:
+- Make a folder for SteamCMD.
+- Inside of it download [***SteamCMD***](https://steamcdn-a.akamaihd.net/client/installer/steamcmd.zip), and unzip it. 
+- Start steamcmd.exe and wait until you got 'Steam>' waiting for your input.
+
+## Getting CSS files
+
+Inside SteamCMD, type or paste each of the following line in the order:
+- `login anonymous`
+- `force_install_dir ./css`
+- `app_update 232330 validate`
+
+## Installing CSS files
+
+- Inside your css folder move the folder `cstrike` at the root of the momentum folder.
```

Comment:
There are a lot of other resources in the `cstrike` folder that we don't need here. I would just dump the needed VPKs into the momentum folder and add `game momentum/cstrike_pak_dir.vpk` to the gameinfo.
![image](https://user-images.githubusercontent.com/9014762/104833982-74329780-5851-11eb-89ba-b6455cc9cf3f.png)

Just trying to get around including more files than we need to.

Reply:
I would chime in here and say that the best place for it would most likely be in a `momentum/custom/` folder, like `momentum/custom/cstrikesource/`

Reply:
yea you'r right about `cstrike`.

But I think those files need to be placed insinde there own folder not just dump into the momentum one.
maybe `momentum/dependencies`  ? 

Gocnak `momentum/custom/cstrikesource/` seems nice too.